examples/voltage_clamp/voltage_clamp.ipynb

```diff
@@ -0,0 +1,801 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "efel.reset()\n",
+    "stim_v = list(range(-80, 40, 10))\n",
+    "feature_names = [\"deactivation_time_constant\"]\n",
+    "feats = efel.get_feature_values(traces, feature_names)\n",
+    "\n",
+    "deact_tau_efel = np.array([feat_dict[\"deactivation_time_constant\"][0] for feat_dict in feats])\n",
+    "plt.plot(stim_v, deact_tau_efel, '.')\n",
+    "plt.xlabel(\"Stimulus Voltage (mV)\")\n",
+    "plt.ylabel(\"Deactivation time constant (ms)\")\n",
+    "plt.title(\"Deactivation time constant\")\n",
+    "plt.xlim((-65, 15))\n",
+    "plt.ylim((-10, 200))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inactivation traces\n",
+    "\n",
+    "Now we will also perform efeature extraction on traces from inactivation experiment. First, we have load the data from the file. Once again, we will put the current data under the 'V' key representing the voltage, so that efel treats our trace like any voltage trace. For this one, we will have to extract the features in two times, because they depend on which stimulus interval we are interested in. We will first look into the features we can extract from the stimulus interval between 1600 and 1700 ms."
```

Comment:
we have load -> we load
in two times -> in two steps ?

Reply:
Fixed in latest commit